model/riscv_pmp_control.sail

```diff
@@ -104,15 +119,52 @@ function pmpCheckRWX(ent, acc) = {
   }
 }
 
-// this needs to be called with the effective current privilege.
-val pmpCheckPerms: (Pmpcfg_ent, AccessType(ext_access_type), Privilege) -> bool
+/* this needs to be called with the effective current privilege.*/
+val pmpCheckPerms: (Pmpcfg_ent, AccessType(ext_access_type), Privilege) -> bool effect {rreg}
 function pmpCheckPerms(ent, acc, priv) = {
-  match priv {
-    Machine => if   pmpLocked(ent)
+  if mseccfg.MML() == 0b1
+  then
+  {
+      if (ent.R() == 0b0 & ent.W() == 0b1)
```

Comment:
Don't put parentheses when you don't need them, if x doesn't need any

Reply:
Removed unnecessary parentheses